packages/macro/index.d.ts

```diff
@@ -216,12 +216,10 @@ export function defineMessage(
  */
 export const msg: typeof defineMessage
 
-type CommonProps = {
+type CommonProps = TransRenderCallbackOrComponent & {
   id?: string
   comment?: string
   context?: string
-  render?: (props: TransRenderProps) => ReactElement<any, any> | null
-  i18n?: I18n
```

Comment:
this was added in https://github.com/lingui/js-lingui/pull/1308

I tested it and (1) doesn't work (2) isn't documented and right now it's easier to remove it

Reply:
Yes, i also spotted it there, and wrote an issue to clatify this https://github.com/lingui/js-lingui/issues/1433